lib/cli/scripts/scan-env.ts

```diff
@@ -69,6 +73,18 @@ async function getPeopleCount(skipCount: number = 0): Promise<PeopleTally> {
     }
 }
 
+async function getHomeFoldersCount(): Promise<number> {
+    try {
+        const nodesApi = new NodesApi(jsApiConnection);
+        const rootApiResult = await nodesApi.listNodeChildren('-root-');
+        const userHomesFolderId = rootApiResult.list.entries.find(associationEntry => associationEntry.entry.name === 'User Homes').entry.id;
+        const homesFolderApiResult = await nodesApi.listNodeChildren(userHomesFolderId);
```

Comment:
On second thought, let's chain the API usage for a more fluent approach. 
You can take [this](https://github.com/Alfresco/alfresco-digital-workspace-app/blob/develop/scripts%2Fci%2Fcheck-env%2Fcleanup-gs-env.js#L67-L73) as an example. 

Hence instead of a style like:
```
const peopleApi = new PeopleApi(jsApiConnection);
const apiResult = await peopleApi.listPeople({
```

it's better to use something chained like for example: 

```
const groups = await alfrescoJsApi.core.groupsApi.getGroups(opt);
```

Reply:
All the `alfrescoJsApi` members will be deprecated in the future (because it prevents proper tree-shaking and optimizations), the first usage is more accurate. I suggest to avoid using any `alfrescoJsApi.XXX` methods for now

Reply:
Good to know @DenysVuika ! Let's go with the former one then, @TomikaArome